cli/src/cortx_prep.sh

```diff
@@ -75,8 +75,8 @@ EOF
 cat <<EOF >/etc/pip.conf
 [global]
 timeout: 60
-index-url: $CORTX_RELEASE_REPO/python_deps/
-trusted-host: $CORTX_RELEASE_REPO
+index-url: file:///tmp/iso_mount/cortx/python_deps/
+trusted-host: file:///tmp/iso_mount/cortx
```

Comment:
Remove `trusted-host` line.
Not required for localhost.

Reply:
addressed.